.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: R
 cache: packages
 
-r_github_packages: 
-- metrumresearchgroup/covrpage   
+apt_packages:
+  - libharfbuzz-dev
+  - libfribidi-dev
+
+r_github_packages:
+- metrumresearchgroup/covrpage
```

Comment:
covrpage has moved to yonicd/covrpage

Reply:
Good point. I should probably be testing that with the CRAN version, though.

Reply:
There’s a CRAN version?

Reply:
Ah, disregard.